cockatrice/src/dlg_manage_sets.cpp

```diff
@@ -128,16 +129,17 @@ WndSets::WndSets(QWidget *parent) : QMainWindow(parent)
     labNotes->setWordWrap(true);
     labNotes->setTextInteractionFlags(Qt::TextBrowserInteraction);
     labNotes->setOpenExternalLinks(true);
-    labNotes->setText("<b>" + tr("Deck Editor") + ":</b> " +
-                      tr("Only cards in enabled sets will appear in the card list of the deck editor") + "<br><br>" +
-                      "<b>" + tr("Card Art") + ":</b> " + tr("Image priority is decided in the following order") +
-                      "<ol><li>" + tr("CUSTOM Folder") +
+    labNotes->setText(tr("Use ctrl+a to select all sets in the view.") + "<br><b>" + tr("Deck Editor") + ":</b> " +
```

Comment:
```suggestion
    labNotes->setText(tr("Use CTRL+A to select all sets in the view.") + "<br><b>" + tr("Deck Editor") + ":</b> " +
```

<br>

Regarding all hintsGrid:
Don't you think the extra line breaks were useful?
Because of how lists are displayed you now have two blocks, but the second block being bullet points of the last element of the first block. The connection and context is not as clear now as seen in your screen.

Yours:
![](https://user-images.githubusercontent.com/36401181/112248786-d5625700-8c56-11eb-820d-de51dd4e340e.png)

Compared to what is currently in client:
![hints](https://user-images.githubusercontent.com/9874850/112284528-a2858680-8c89-11eb-9f08-b89d6b6ccc4b.png)

Reply:
I wanted to make it smaller, it's taking up a lot of space

Reply:
I see, sure. Both not ideal. :)